cmd/kubeadm/app/util/patches/patches.go

```diff
@@ -322,17 +323,17 @@ func getPatchSetsFromPath(targetPath string, knownTargets []string, output io.Wr
 		goto return_path_error
 	}
 
-	err = filepath.Walk(targetPath, func(path string, info os.FileInfo, err error) error {
+	err = filepath.WalkDir(targetPath, func(path string, d fs.DirEntry, err error) error {
```

Comment:
you seem to have 2 prs
my comment about this
https://github.com/kubernetes/kubernetes/pull/129944#discussion_r1940783220